Pipfile

```diff
@@ -80,12 +79,18 @@ flask = "==2.3.2"
 certifi = "*"
 Markdown = "*"
 dc-django-utils = {file = "https://github.com/DemocracyClub/dc_django_utils/archive/refs/tags/7.0.1.tar.gz"}
+playwright = "==1.48.0"
+pytest-django = "*"
+pytest = "*"
+pytest-playwright = "*"
+pytest-cov = "*"
+moto = "*"
```

Comment:
One of the things I was hoping we would cover under the banner of "removing unused code[/stuff]" was unused packages. I think there are 2 big problems we have on this project:

1. I am convinced there is a bunch of stuff in the package tree that we actually don't use at all any more
2. There is no seperation of prod and dev packages. This is especially problematic for an application deployed to lambda because every time we do a cold boot we are spending time unzipping testing/dev packages that are not used at runtime. Specifically here you are adding playwright, pytest, moto etc into the stuff we have to unzip when we cold boot.

I'd really like us to fix both of these problems.
If the next thing you are going to do is migrate from pipenv to UV, I am happy to just ignore this for now and include it in the UV migration PR. This was one of the things I did as part of that exercise on EE:
- https://github.com/DemocracyClub/EveryElection/pull/2282
- https://github.com/DemocracyClub/EveryElection/pull/2282/commits/6d5d0d10addc24f3c143b49a75b5219db63794b1
- https://github.com/DemocracyClub/EveryElection/pull/2282/commits/7ad181a819f1e1e247b6b39026910f505560e4de

Although the scale of the problem is bigger on this project.

Anyway. If we want to just ignore it for now, fine. Just want to flag it as a thing that needs doing.